pallets/did/src/tests.rs

```diff
@@ -2352,17 +2384,18 @@ fn check_invalid_signature_operation_verification() {
 	);
 	let signature = alternative_key.sign(&call_operation.encode());
 
-	let mut ext = ExtBuilder::default().with_dids(vec![(did, mock_did)]).build(None);
-
-	ext.execute_with(|| {
-		assert_noop!(
-			Did::verify_did_operation_signature_and_increase_nonce(
-				&call_operation,
-				&did::DidSignature::from(signature)
-			),
-			did::DidError::SignatureError(did::SignatureError::InvalidSignature)
-		);
-	});
+	ExtBuilder::default()
+		.with_dids(vec![(did, mock_did)])
+		.build(None)
+		.execute_with(|| {
```

Comment:
I really like this short version. I guess you updated all of them now? 😅 I would have only done the once you really had to change anyways but fine. 👍🏻